docs/tutorials/advanced-autodiff.md

```diff
@@ -1,15 +1,1846 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.15.2
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
 (advanced-autodiff)=
-# Advanced automatic differentiation
+# Advanced automatic differentiation 201
+
+In this tutorial, you will learn about complex applications of automatic differentiation (autodiff) in JAX and gain a better understanding of how taking derivatives in JAX can be both easy and powerful.docs.g
+
+Make sure to check out the {ref}`automatic-differentiation` tutorial to go over the JAX autodiff basics, if you haven't already.
+
+This tutorial covers the following topics:
 
-```{note}
-This is a placeholder for a section in the new {ref}`jax-tutorials`.
+- {ref}`advanced-autodiff-taking-gradients-part-2`
+  - {ref}`advanced-autodiff-higher-order-derivatives`
+  - {ref}`advanced-autodiff-higher-order-optimization`
+  - {ref}`advanced-autodiff-straight-through-estimator-using-stop_gradient`
+  - {ref}`advanced-autodiff-per-example-gradients`
+  - {ref}`advanced-autodiff-hessian-vector-products-with-jax-grad-of-jax-grad`
+  - {ref}`advanced-autodiff-jacobians-and-hessians`
+- {ref}`advanced-autodiff-how-autodiff-functions-are-made`
+  - {ref}`advanced-autodiff-jacobian-vector-product`
+  - {ref}`advanced-autodiff-vector-jacobian-product`
+  - {ref}`advanced-autodiff-vector-valued-gradients-with-vjps`
+  - {ref}`advanced-autodiff-hessian-vector-products-using-forward-and-reverse`
+- {ref}`advanced-autodiff-composing-vjps-jvps-and-jax-vmap`
+  - {ref}`advanced-autodiff-jacobian-matrix-and-matrix-jacobian-products`
+  - {ref}`advanced-autodiff-implementation-of-jax-jacfwd-and-jax-jacrev`
+- {ref}`advanced-autodiff-complex-numbers-and-differentiation`
+- {ref}`advanced-autodiff-custom-derivative-rules`
+  - {ref}`advanced-autodiff-custom-derivative-rules-example-problems`
+  - {ref}`advanced-autodiff-basic-usage-jax-custom-jvp-jax-custom-vjp-apis`
+  - {ref}`advanced-autodiff-custom-derivative-rules-more-features-and-details`
```

Comment:
I'd avoid putting this TOC here – it's likely to get out of sync if and when the doc is updated, and the built docs will automatically generate a table of contents in the sidebar.

Reply:
Done